### PR TITLE
[Carbon Black Enterprise Response] Alert Name Changes

### PR DESCRIPTION
**Pull request by ennozdd**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
The integration now creates alerts with more meaningful names.

Before the change:
Carbon Black EDR - 000000d9-0000-4424-01d8-e785b78b7857-01840934aac5 - powershell.exe

After the change:
EDR Alert: Execution - Command and Scripting Interpreter Execution -- powershell.exe

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Carbon_Black_Enterprise_Response/Integrations/CarbonBlackResponseV2/CarbonBlackResponseV2.py [hunk 1]
```diff
@@ -855,8 +855,14 @@ def fetch_incidents(client: Client, max_results: int, last_run: dict, first_fetc
                 continue
 
         alert_id = alert.get('unique_id', '')
-        alert_name = alert.get('process_name', '')
-        incident_name = f'{INTEGRATION_NAME}: {alert_id} {alert_name}'
+        process_name = alert.get('process_name','')
+        if alert.get('feed_name', '') == 'My Watchlists':
+            alert['description'] = alert.get('watchlist_name', '')
+        alert_name = alert.get('description', '')
```
Comment (ennozdd) [inline Packs/Carbon_Black_Enterprise_Response/Integrations/CarbonBlackResponseV2/CarbonBlackResponseV2.py:861]: custom watchlist names are stored in "watchlist_name" and feeds coming from other sources store their names in the "description" field

**Comment by JasBeilin** [comment]
Hi @ennozdd , thank you for this contribution!
Can you please share the documentation which this is taken from?
As far as we know, the name of the alert is process by design, before we change something this big, we would like to further understand this change.

**Comment by JasBeilin** [comment]
Hi @ennozdd, please help us in moving forward here. This is a sensitive change and we cannot make progress without further understanding.

**Comment by ennozdd** [comment]
hi @JasBeilin the design is not documented so I relied on my observations for the changes.

**Comment by JasBeilin** [comment]
Hi @ennozdd,
Although we greatly appreciate your contribution, it has been rejected for the following reason: 
Breaking backward compatibility while it can be done using other method (mapper/ pre-process rule/ Playbook automation).

Thank you for understanding.
We look forward to receiving more contributions from you in the future.

**Pull request closed without merging**